sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-energy-demand/automl-forecasting-task-energy-demand-advanced-mlflow.ipynb

```diff
@@ -148,76 +148,135 @@
+    "cluster_name = \"energy-cluster-v2\"\n",
     "\n",
-    "- trial_timeout_minutes - Maximum time in minutes that each trial (child job) can run for before it terminates. If not specified, a value of 1 month or 43200 minutes is used.\n",
-    "    \n",
-    "- max_trials - The maximum number of trials/runs each with a different combination of algorithm and hyperparameters to try during an AutoML job. If not specified, the default is 1000 trials. If using 'enable_early_termination' the number of trials used can be smaller.\n",
-    "    \n",
-    "- max_concurrent_trials - Represents the maximum number of trials (children jobs) that would be executed in parallel. It's a good practice to match this number with the number of nodes your cluster.\n",
-    "    \n",
-    "- enable_early_termination - Whether to enable early termination if the score is not improving in the short term. \n",
-    "    "
+    "try:\n",
+    "    # Retrieve an already attached Azure Machine Learning Compute.\n",
+    "    compute = ml_client.compute.get(cluster_name)\n",
+    "except ResourceNotFoundError as e:\n",
+    "    compute = AmlCompute(\n",
+    "        name=cluster_name,\n",
+    "        size=\"STANDARD_DS12_V2\",\n",
+    "        type=\"amlcompute\",\n",
+    "        min_instances=0,\n",
+    "        max_instances=4,\n",
+    "        idle_time_before_scale_down=120,\n",
+    "    )\n",
+    "    poller = ml_client.begin_create_or_update(compute)\n",
+    "    poller.wait()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Specialized Forecasting Parameters\n",
+    "# 4. Configure and run the AutoML Forecasting training job\n",
+    "In this section we will configure and run the AutoML job to train the model.\n",
+    "\n",
+    "## 4.1 Configure the job through the forecasting() factory function\n",
+    "\n",
+    "### forecasting() function parameters:\n",
+    "\n",
+    "The `forecasting()` factory function allows user to configure AutoML for the forecasting task for the most common scenarios with the following properties.\n",
+    "\n",
+    "|Property|Description|\n",
+    "|-|-|\n",
+    "|**target_column_name**|The name of the label column.|\n",
+    "|**primary_metric**|This is the metric that you want to optimize.<br> Forecasting supports the following primary metrics <br><i>spearman_correlation</i><br><i>normalized_root_mean_squared_error</i><br><i>r2_score</i><br><i>normalized_mean_absolute_error</i>|\n",
+    "|**training_data**|The training data to be used for this experiment. You can use a registered MLTable in the workspace using the format `<mltable_name>:<version\\>` OR you can use a local file or folder as a MLTable. For e.g Input(mltable='my_mltable:1') OR Input(mltable=MLTable(local_path=\"./data\")) The parameter 'training_data' must always be provided.|\n",
```

Comment:
- The `<mltable_name>:<version\>` renders an extra "\". Please remove it.
- Please replace "Input(mltable='my_mltable:1')" with `Input(mltable='my_mltable:1')`
- Please replace "Input(mltable=MLTable(local_path=\"./data\"))" with `Input(mltable=MLTable(local_path="./data"))`

Reply:
Fixed.